Sources/SmartStoreWrapper/SmartStoreWrapper.swift

```diff
@@ -0,0 +1,11 @@
+// SmartStoreWrapper.swift
+// Re-exports SmartStore framework along with its dependencies
+
+// Re-export the SmartStore framework
+@_exported import SmartStore
+
+// Re-export SQLCipher for encryption capabilities
```

Comment:
We don't really need to re-export SQLCipher and FMDB but that way consuming apps can use those APIs if they wish to.